aws/offsite/lambda/verify-database-copy/index.js

```diff
@@ -0,0 +1,114 @@
+console.log('Loading function');
+
+const AWS = require('aws-sdk');
+const mysqlPromise = require('promise-mysql');
+const Slack = require('slack-node');
+const promisify = require('util-promisify');
+const setTimeoutPromise = promisify(setTimeout);
+
+const SLACK_WEBHOOK_URL = `https://hooks.slack.com/services/${process.env.SLACK_WEBHOOK_URL}`;
+const SLACK_CHANNEL = process.env.SLACK_CHANNEL;
+
+const DB_INSTANCE_IDENTIFIER = process.env.DB_INSTANCE_IDENTIFIER;
+const DB_HOST = process.env.DB_HOST;
+const DB_NAME = process.env.DB_NAME;
+const DB_USER = process.env.DB_USER;
+const DB_PASSWORD = process.env.DB_PASSWORD;
+
+var status_message = '';
+
+
+var postStatusToSlack = function postStatusToSlack(message) {
+    console.log('Posting status to Slack');
+    var slack = new Slack();
```

Comment:
Our javascript style guide says to use 2-space indents. rather than fixing a bunch of style issues manually though, what we really need to do is get eslint running on the files in this directory. It seems like you'd want to start with creating a /aws/offsite/lambda/package.json which defines a `lint` command similar to this one: https://github.com/code-dot-org/code-dot-org/blob/e02827442a98cefd4b17edcc7d0d1455d1b8579c/apps/package.json#L11

Then you'd need to change this rake task to also run lint in this directory: https://github.com/code-dot-org/code-dot-org/blob/e02827442a98cefd4b17edcc7d0d1455d1b8579c/lib/rake/lint.rake#L21-L26

by default, this would pull in the root .eslintrc.js but not apps/.eslintrc, so you would probably also want to add a /aws/offsite/lambda/.eslintrc (modeled off the one in apps) to at least allow es6 syntax.

Given the size of this project it seems ok to do in a separate PR.